opensearch-operator/Dockerfile-ARM

```
@@ -0,0 +1,30 @@
+# Build the manager binary
+FROM golang:1.17 as builder
+
+WORKDIR /workspace
+# Copy the Go Modules manifests
+COPY go.mod go.mod
+COPY go.sum go.sum
+# cache deps before building and copying source so that we don't need to re-download as much
+# and so that source changes don't invalidate our downloaded layer
+RUN go mod download
+
+# Copy the go source
+COPY main.go main.go
+COPY api/ api/
+COPY controllers/ controllers/
+COPY pkg/   pkg/
+COPY opensearch-gateway/   opensearch-gateway/
+
+# Build
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o manager main.go
```

By any chance you meant `arm64`?
```suggestion
RUN CGO_ENABLED=0 GOOS=linux GOARCH=arm64 go build -a -o manager main.go
```

---

Of-course! Thank you, will be fix right away